### PR TITLE
Add iOS.10 with arm support to RID graph

### LINKED ISSUE #33982: Add RID for 32bit ios
This would be straight forward if Apple didn't drop the support for 32bits in the recent versions. The last supported 32bit version is ios 10.

RIDs for other ios architectures already exist and only 32 bit needs to be added up to `ios-10-arm`.

@ericstj please advice on the correct logic.

### PR DESCRIPTION
**Pull request by ericstj**
Fixes #33982

### CODE REVIEW

**Review by stephentoub — APPROVED** [review]

File: src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props [hunk 1]
```diff
@@ -57,11 +57,15 @@
       <Architectures>x64</Architectures>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="ios">
+      <Parent>unix</Parent>
+      <Architectures>arm</Architectures>
+      <Versions>10</Versions>
```
Comment (marek-safar) [inline src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props]:
```suggestion
      <Versions>8;9;10</Versions>
```

File: src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props [hunk 2]
```diff
@@ -57,11 +57,15 @@
...
+    </RuntimeGroup>
     <RuntimeGroup Include="ios">
       <Parent>unix</Parent>
       <Architectures>arm64;x64</Architectures>
-      <!-- starting from 11 because 10 will need different graph with arm included -->
-      <Versions>11;12;13</Versions>
+      <Versions>10;11;12;13</Versions>
```
Comment (marek-safar) [inline src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props]:
```suggestion
      <Versions>8;9;10;11;12;13</Versions>
```

File: src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props (hunk 1 above)
Comment (akoeplinger) [reply src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props]: We actually go back to iOS 7 for ARM/ARM64.

Comment (ericstj) [reply src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props]: What about x64?  Marek's suggestion below indicates that's also supported back to 7.

Comment (marek-safar) [reply src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props]: Let's start at 8 everywhere

**Review by akoeplinger — APPROVED** [review]

**Review by marek-safar — APPROVED** [review]

**Pull request merged**